Shenhe-Ayaka Banner added to the simulator

ScarletSeal opened the pull request:
Let me know if there's anything I need to change :)
Changelog:
-Added the new banners (Ayaka, Shenhe, Epitome-44)
-Added images, as webp, for the banners and the thumbnails
-Moved the text "lonesome transcendence" so it matches with the banner image
-Implemented Mika in every language that supports items.
-Opening the site sends you to Ayaka's banner straight away

AguzzTN54 commented on src/lib/data/banners/events/3.5.json:
```diff
@@ -69,6 +69,71 @@
+							"character": "shenhe",
+							"buttonBoxPosition": {
+								"w": 160,
+								"t": 22
```
Because this character banner already existed before, it's better to use the same settings I've applied.

https://github.com/AguzzTN54/Genshin-Impact-Wish-Simulator/blob/36fe7a881a311dd75cd37df1ffff6e7c191becc9/src/lib/data/banners/events/2.4.json#L12-L17

AguzzTN54 commented on src/lib/data/banners/events/3.5.json:
```diff
@@ -69,6 +69,71 @@
+							"buttonBoxPosition": {
+								"w": 95,
+								"t": 32,
+								"l": 50
```
https://github.com/AguzzTN54/Genshin-Impact-Wish-Simulator/blob/36fe7a881a311dd75cd37df1ffff6e7c191becc9/src/lib/data/banners/events/2.6.json#L83-L87

AguzzTN54 commented on src/lib/data/banners/events/3.5.json:
```diff
@@ -69,6 +69,71 @@
+								"w": 80,
+								"l": 35
+							}
+						}
```
The Weapon button as well, you can add more number for calamity to set the icon on the right side.

https://github.com/AguzzTN54/Genshin-Impact-Wish-Simulator/blob/36fe7a881a311dd75cd37df1ffff6e7c191becc9/src/lib/data/banners/events/2.4.json#L44-L50

https://github.com/AguzzTN54/Genshin-Impact-Wish-Simulator/blob/36fe7a881a311dd75cd37df1ffff6e7c191becc9/src/lib/data/banners/events/2.0.json#L28-L34

AguzzTN54 commented on src/lib/data/wish-setup.json:
```diff
@@ -1,6 +1,6 @@
 	"version": "3.5",
-	"wishPhase": 1,
+	"wishPhase": 2,
 	"storageVersion": "3.5.0",
```
`storageVersion` needs to be changed to automatically apply updates to site.

AguzzTN54 replied:
Yeah, it's okay.
Actually it doesn't matter if you want to change it to any number, the site will match with the stored data, if this property does not change, the site will not update automatically.

AguzzTN54 reviewed (changes requested):
Mostly looks great, but there are a few that are missing.
And don't forget to add Mika's splash art too.

AguzzTN54 commented:
![image](https://user-images.githubusercontent.com/13815468/225620981-423ede7f-624a-4167-a7fe-058e9c759f0d.png)

And one more thing, I hope you can explain what changes you made to each commit to make it easier to manage the code, not just random messages like this.

AguzzTN54 commented on src/lib/data/banners/events/3.5.json:
```diff
@@ -69,6 +69,71 @@
+							}
+						}
+					],
+					"rateup": ["the-bell", "favonius-warbow", "sacrificial-sword", "wine-and-song", "favonius-lance"],
```
The first order of the rateup weapons is Wine and Song

![image](https://user-images.githubusercontent.com/13815468/225625657-3f44ace9-563f-4452-b671-839603de2eef.png)

ScarletSeal commented:
-Shenhe, Ayaka, Mistsplitter buttonBox were copied 1:1 as you requested
-Queller buttonBox was copied as well, but also moved 30 units to the left so that it appears next to Mistsplitter
-Added Mika splashes in their respective folders
-Wine and Song now appears as the first weapon instead of The Bell
-storageVersion 3.5.0->3.5.1

I named the new commits, and added descriptions for easier readability. Old commits are a mess because I deleted my old repository that time, so I was just trying to get back to where I was before deletion.

AguzzTN54 commented on src/lib/data/banners/events/3.5.json:
```diff
@@ -69,6 +69,71 @@
+								"t": 32
+							},
+							"textOffset": {
+								"l": 51
```
![OnPaste 20230316-202917](https://user-images.githubusercontent.com/13815468/225632151-1bababa2-7918-4c3d-8bbc-213edafc222e.jpg)

Oh, I forgot something, this text should be left aligned with the stars.

ScarletSeal replied:
![image](https://user-images.githubusercontent.com/118117643/225640881-05dfe3a6-13af-42fc-a4a9-4e58341d6b09.png)
![image](https://user-images.githubusercontent.com/118117643/225640922-51ae96be-7503-42b8-baaf-dea657eed6e6.png)

This is what they look like now. I updated the text offset on both banners

AguzzTN54 commented:
After double checking, I just realized that Shenhe's banner has different aspect ratio from the others and webpage screenshot still not updated yet. I have fixed these issues as I update the changelog.
I've pushed to the production server, as far as I can tell everything is running fine right now because of your help.

Thanks for your hardworking, I really appreciate.
